### ISSUE TITLE
TimeInput - name attribute accessibility violation WCAG 1.1.1 and 4.1.2

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#8134]
An input should have a name associated with it. Currently the name is getting put on the hidden input within TimeInput. It should instead be placed on the regular input so that it is available to screen reader users.

Related WCAG Rule
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html

**Comment by jcfilben** [comment]
Did more research here. The WCAG rule requires that UI elements that a user can see or interact with have a name/role/value.

In this case the interactive and visible part of the component is the spinbutton group, which meets the name/role/value requirement.
The hidden input where the name lands has type='hidden' so this wcag rule doesn't apply
